src/main/java/no/finn/unleash/Unleash.java

```diff
@@ -15,6 +15,10 @@ default boolean isEnabled(String toggleName, UnleashContext context, boolean def
         return isEnabled(toggleName, defaultSetting);
     }
 
+    boolean isEnabled(final String toggleName, boolean defaultSetting, final FallbackAction fallbackAction);
```

Comment:
Instead of defining a new interface we could actually use one of the functional interfaces: `BiFunction<String, UnleashContext, Boolean>` instead.

Reply:
Oh yeah, completely forgot about these, I will change that accordingly.